recipes/jupyterlab-nbconvert-nocode/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "jupyterlab_nbconvert_nocode" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
```

```suggestion
  name: {{ name|lower }}-meta
```
This is a meta-package and will not actually be built.